src/app/pages/RewardPage/components/RewardsDetail/index.tsx

```diff
@@ -84,7 +84,16 @@ export const RewardsDetail: React.FC<IRewardsDetailProps> = ({
           <LoadableValue
             value={
               bignumber(availableAmount).greaterThan(0) ? (
-                <Tooltip content={`${availableAmount} ${asset}`}>
+                <Tooltip
```

Comment:
You don't need this `Tooltip`, just use `useTooltip` in `AssetValue` component and it will be handled automatically.